BeSpoke/Views/SharedViews/LoginView.swift

```diff
@@ -0,0 +1,76 @@
+//
+// CreateOrLoginView.swift
+// bespoke
+//
+// Created by Jared Webber on 2023-09-25
+//
+
+import AuthenticationServices
+import FirebaseAuth
+import FirebaseCore
+import GoogleSignInSwift
+import SwiftUI
+
+struct LoginView: View {
+    @Environment(\.colorScheme) var colorScheme
+    @Environment(\.dismiss) var dismiss
+
+    @EnvironmentObject var authManager: AuthManager
+
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 16) {
+                Spacer()
+                Image("loginScreen")
+                    .foregroundStyle(Color("blue"))
+                    .padding()
+                Spacer()
+
+                SignInWithAppleButton(
+                    onRequest: { _ in
+                        print("Sign In With Apple")
+                        // TODO: Request Apple Authorization
+                    },
+                    onCompletion: { _ in
+                        // TODO: Handle AppleID Completion
+                    }
+                )
+                .signInWithAppleButtonStyle(colorScheme == .light ? .black : .white)
+                .frame(width: 280, height: 45, alignment: .center)
+
+                GoogleSignInButton {
+                    signInWithGoogle()
+                    print("Sign In With Google")
```

Comment:
Nit: Is this just for logging purposes? Maybe "Signed in with Google" then?

Reply:
Thats just leftover from before the button did anything we can get rid of it I think